inc/classes/preload/class-sitemap.php

```diff
@@ -195,4 +259,56 @@ public function process_sitemap( $sitemap_url, $urls = [] ) {
 		$urls = array_merge( $urls, $tmp_urls );
 		return $urls;
 	}
+	
+	/**
+	 * Get URLs from WordPress
+	 * 
+	 * Used as a fallback when extracting URLs from sitemap fails.
+	 *
+	 * @since 3.2.5
+	 * @author Arun Basil Lal
+	 * 
+	 * @link https://github.com/wp-media/wp-rocket/issues/1306
+	 *
+	 * @return array $urls Array of permalinks.
+	 */
+	public function get_urls() {
+		
+		$urls = [];
+		
+		// Get public post types
+		$post_types = get_post_types( array( 'public' => true ) );
+		$post_types = array_filter( $post_types, 'is_post_type_viewable' );
+		
+		/**
+		 * Filters the arguments for get_posts
+		 *
+		 * @since 3.2.4
+		 * @author Arun Basil Lal
+		 *
+		 * @param array $args Arguments for get_posts
+		 */
+		$args = apply_filters(
+			'rocket_preload_sitemap_fallback_request_args',
+			[
+				'fields'			=> 'ids',
+				'numberposts' 		=> 1000,
+				'posts_per_page'  	=> -1,
+				'post_type'			=> $post_types,
```

Comment:
aligment issue

Reply:
Alignment looks correct here. But not here 🤔 - https://github.com/wp-media/wp-rocket/blob/feature/1306-sitemap-preload-fallback/inc/classes/preload/class-sitemap.php